net/haproxy/src/opnsense/service/templates/OPNsense/HAProxy/haproxy.conf

```diff
@@ -1032,6 +1032,9 @@ backend {{backend.name}}
     {{customOpt}}
 {%         endfor %}
 {%       endif %}
+{%       if backend.tuning_httpreuse|default("") != "" %}
+    http-reuse {{backend.tuning_httpreuse}}
```

Comment:
This will add the `http-reuse` options also in TCP backends. Would you please add a if-clause to ensure it is only added for HTTP backends?